### PR TITLE
PR: Update procedure to connect to external kernels for Spyder 3.3

### LINKED ISSUE #53: Update instructions to connect to an external kernel
After https://github.com/spyder-ide/spyder/pull/7306, instead of running `ipykernel start` you need to run `python -m spyder_kernels.console`.

### PR DESCRIPTION
**Pull request by CAM-Gerlach**
### Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 3.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below
* [x] Noted what issue(s) this pull request resolves, creating one if needed

## Description of Changes

Updates the section describing connecting to an externel IPython kernel with Spyder to reflect the newly modular ``spyder-kernels`` package in Spyder >=3.3.0. Please review for correctness.

### Issue(s) Resolved

Fixes #53

### CODE REVIEW

File: doc/ipythonconsole.rst [hunk 1]
```diff
@@ -39,7 +39,9 @@ Connect to an external kernel
 
 To connect to an external kernel,
 
-#. Launch an IPython kernel on the local or remote host if one is not already running, e.g. with ``ipython kernel``.
+#. Launch an IPython kernel on the local or remote host if one is not already running.
+   If using Spyder 3.3.0 or later or Spyder 4, you'll want to do so with ``python -m spyder_kernels.console`` (after you've first installed ``spyder-kernels`` on the host with ``<conda/pip> install spyder-kernels``).
```
Comment (ccordoba12) [inline doc/ipythonconsole.rst]: Let's remove `Spyder 4` from here because it could confuse users since it's not released yet.

Comment (ccordoba12) [reply doc/ipythonconsole.rst]: Also, I'd prefer to change

> you'll want to do so ...

for

> you need to so ...

Comment (CAM-Gerlach) [reply doc/ipythonconsole.rst]: Done and done.

File: doc/ipythonconsole.rst [hunk 2]
```diff
@@ -39,7 +39,9 @@ Connect to an external kernel
...
+   If using a version of Spyder before 3.3.0, ``ipython kernel`` should work to launch the kernel, albeit without certain Spyder-specific features.
```
[the remark this answers is not in the document]
Comment (CAM-Gerlach) [reply doc/ipythonconsole.rst]: But aren't they already installed using the given ``pip``/``conda`` command, if they are indeed dependencies? They were when I tested it using the former...(at least the first two).

**Pull request merged**